web/migrations/0049_learningmap_learningmapnode.py

```
@@ -0,0 +1,101 @@
+                        ],
+                        default="blue-600",
+                        max_length=20,
+                    ),
+                ),
+                ("x_position", models.IntegerField(default=0)),
+                ("y_position", models.IntegerField(default=0)),
+                ("order", models.PositiveIntegerField(default=0)),
+                (
+                    "enrollment",
+                    models.ForeignKey(
+                        blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to="web.enrollment"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)